.github/workflows/benchmark.yml

```diff
@@ -68,7 +68,7 @@ jobs:
   tpcds-1g-gen:
     name: "Generate an input dataset for TPCDSQueryBenchmark with SF=1"
     if: contains(inputs.class, 'TPCDSQueryBenchmark') || contains(inputs.class, '*')
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
In order to avoid uncertain compatibility in the future, I am using 24.04 here instead of latest.